test/e2e/storage/persistent_volumes-local.go

```diff
@@ -307,6 +310,88 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("should support SELinux labeling on local storage volume", func() {
+
+		var testVol *localTestVolume
+
+		BeforeEach(func() {
+			testVol = setupLocalVolumePVCPV(config, DirectoryLocalVolumeType)
```

Comment:
I would also test this with the mount type, following the similar format as the other tests above.